testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -243,6 +243,26 @@ def test_check_coordinate_limits_max(self):
         assert_raises(ValueError, u.atoms.write, self.outfile)
         del u
 
+    @attr('issue')
+    def test_check_header_title_multiframe(self):
+        """Check whether HEADER and TITLE are written just once in a multi-
+        frame PDB file (Issue 741)"""
+        u = mda.Universe(PSF, DCD, permissive=True)
+        protein = u.select_atoms("protein and name CA")
+        pdb = mda.Writer(self.outfile, multiframe=True)
+        for ts in u.trajectory[:5]:
+            pdb.write(u)
+        pdb.close()
+        u2 = mda.Universe(self.outfile)
+        if u2.trajectory.header:
```

Comment:
This test can't check if the `HEADER` record appears multiple times in a PDB. The entry in that member.variable is always a string. You can check with this [test.pdb](https://gist.github.com/kain88-de/1626b3a3d680cde8fb69) that `trajectory.header` has only one entry.